src/PostgREST/ApiRequest/Types.hs

```diff
@@ -229,6 +229,7 @@ data SimpleOperator
   | OpAdjacent
   | OpMatch
   | OpIMatch
+  | OpIsDistinctFrom
```

Comment:
One thing here. So on https://github.com/PostgREST/postgrest/pull/2738 I'm implementing any/all modifiers for operators, and I've realized this fits in pretty well with the `SimpleOperator` types.. because they all can use them like:

```sql
select 1 = any('{3,2,1}');
select 1 > any('{3,2,1}');
```

However `IS DISTINCT FROM` cannot be used in that way.

Could you make it instead be a part of `Operation`? Like:

```haskell
data Operation
  = Op SimpleOperator (Maybe OpQuantifier) SingleVal
  | In ListVal
  | Is TrileanVal
  | IsDistinctFrom SingleVal
  | Fts FtsOperator (Maybe Language) SingleVal
  deriving (Eq)
```